modules/internal/ChapelBase.chpl

```diff
@@ -1488,6 +1488,82 @@ module ChapelBase {
       throw new owned TaskErrors(e.errors);
   }
 
+  // Routines for re-interpreting reals as uints and vice-versa, at
+  // the bit level
+  //
+  @unstable("This routine may change names / signatures")
+  proc param (real(64)).transmute(type t) param : t {
+    if t != uint {
+      compilerError("Cannot (currently) transmute from real(64) to " +
+                    t:string);
```

Comment:
Mildly curious if you could just replace all error message with:

```chpl
compilerError("Cannot (currently) transmute from " + this.type:string + " to " + t:string);
```

to make future error message updates easier